### ISSUE TITLE
Slient failure (exit 0) with Ray after upgrade to v0.12.0

### ISSUE THREAD

**Issue by drikster80** [axolotl-ai-cloud/axolotl#3048]
### Please check that this issue hasn't been reported before.

- [x] I searched previous [Bug Reports](https://github.com/axolotl-ai-cloud/axolotl/labels/bug) didn't find any similar reports.

### Expected Behavior

After running `axolotl train /models/test/14b-fft.yaml --use-ray`, training should start, using Ray for multi-node.

### Current behaviour

The Axolotl logo is displayed and the no other logs are provided. No errors, only the ASCII logo. The training job exits as successful (`exit 0`).

**Using the same training yaml without Ray/FSDP (commented out) on a single node is successful.
**Using the same training yaml with v0.11.0.post1 is successful

NOTE: Environment is **ARM64** on GH200 cluster

### Steps to reproduce

1. Install Axolotl v0.12.0
2. Execute training job using --use-ray

### Config yaml

```yaml
base_model: /models/Qwen3-14B
# Automatically upload checkpoint and final model to HF
# hub_model_id: username/custom_model_name

load_in_8bit: false
load_in_4bit: false
strict: false

plugins:
  - axolotl.integrations.liger.LigerPlugin

liger_rope: true
liger_rms_norm: true
liger_glu_activation: true
liger_layer_norm: true
liger_fused_linear_cross_entropy: true

datasets:
  - path: NousResearch/Hermes-3-Dataset
    type: chat_template
    field_messages: conversations
    message_field_role: from
    message_field_content: value

val_set_size: 0.05

chat_template: chatml

output_dir: /models/Qwen3-14B/outputs/fft

sequence_len: 4096
sample_packing: true
pad_to_sequence_len: true

flash_attention: true

use_wandb: true
wandb_project: patapsco
#wandb_entity:
#wandb_watch:
wandb_name: fft
#wandb_log_model:

gradient_accumulation_steps: 2
micro_batch_size: 8
num_epochs: 2
optimizer: paged_adamw_32bit
lr_scheduler: cosine
learning_rate: 1e-6

bf16: true
tf32: false

resume_from_checkpoint:
logging_steps: 1

evals_per_epoch: 10
saves_per_epoch: 10
save_total_limit: 2

warmup_steps: 60
weight_decay: 0.0
fsdp:
  - full_shard
  - auto_wrap

fsdp_config:
  fsdp_version: 2
  fsdp_offload_params: false
  fsdp_cpu_ram_efficient_loading: true
  fsdp_auto_wrap_policy: TRANSFORMER_BASED_WRAP
  fsdp_transformer_layer_cls_to_wrap: Qwen3DecoderLayer
  fsdp_state_dict_type: FULL_STATE_DICT
  fsdp_sharding_strategy: FULL_SHARD
  fsdp_reshard_after_forward: true
  fsdp_activation_checkpointing: true

special_tokens:

use_ray: true
ray_num_workers: 8
resources_per_worker:
  GPU: 1
```

### Which Operating Systems are you using?

- [x] Linux
- [ ] macOS
- [ ] Windows

### Python Version

3.12

### axolotl branch-commit

v0.12.0

### Acknowledgements

- [x] My issue title is concise, descriptive, and in title casing.
- [x] I have searched the existing issues to make sure this bug has not been reported yet.
- [x] I am using the latest version of axolotl.
- [x] I have provided enough information for the maintainers to reproduce and diagnose the issue.

**Comment by winglian** [comment]
There is a possibility this is the breaking change https://github.com/axolotl-ai-cloud/axolotl/commit/bb1cae1a20dd10bce644319d6fc26b1d51c1d666

we have Ray tests in our CI which are currently passing, but they have limited scopes. https://github.com/axolotl-ai-cloud/axolotl/blob/main/tests/e2e/multigpu/test_ray.py

**Comment by winglian** [comment]
@drikster80 This is resolved in v0.12.1

**Comment by drikster80** [comment]
> [@drikster80](https://github.com/drikster80) This is resolved in v0.12.1

Thanks! Was just logging in to take a look at the comment from yesterday and it looks like it's already fixed!
